[Bug] winGRASS 8.3.dev compilation fails with --with-openmp

**Describe the bug**

**winGRASS 8.3.dev** compilation fails with --with-openmp.

I don't see this in github osgeo4w windows CI actions.

though I see it here on my win10 OSGeo4W/Msys2 winGRASS 8.3.dev build environment, and also at Martin's dailys builds.

e.g. https://wingrass.fsv.cvut.cz/grass83/logs/log-r001bf57c8b-1/error.log

```
GRASS GIS 8.3.dev 001bf57c8b compilation log
--------------------------------------------------
Started compilation: Sun Feb 19 14:06:42 CEST 2023
--
Errors in:
/usr/src/grass83/lib/rst/interp_float
/usr/src/grass83/lib/gpde
/usr/src/grass83/raster/r.gwflow
/usr/src/grass83/raster/r.mfilter
/usr/src/grass83/raster/r.neighbors
/usr/src/grass83/raster/r.patch
/usr/src/grass83/raster/r.resamp.filter
/usr/src/grass83/raster/r.resamp.interp
/usr/src/grass83/raster/r.resamp.rst
/usr/src/grass83/raster/r.series
/usr/src/grass83/raster/r.series.accumulate
/usr/src/grass83/raster/r.sim/simlib
/usr/src/grass83/raster/r.sim/r.sim.water
/usr/src/grass83/raster/r.sim/r.sim.sediment
/usr/src/grass83/raster/r.slope.aspect
/usr/src/grass83/raster/r.solute.transport
/usr/src/grass83/raster/r.sun
/usr/src/grass83/raster/r.univar
/usr/src/grass83/raster3d/r3.gwflow
/usr/src/grass83/vector/v.surf.bspline
/usr/src/grass83/vector/v.surf.rst
--
In case of errors please change into the directory with error and run 'make'.
If you get multiple errors, you need to deal with them in the order they
appear in the error log. If you get an error building a library, you will
also get errors from anything which uses the library.
--
Finished compilation: Sun Feb 19 14:38:52 CEST 2023
```
first failing is always in **lib/rst/interp_float**

disabling by `--with-openmp=no`, compilation works.

a fresh 8.3.dev checkout

```
--------------------------------------------------
Started compilation: Sat Mar 11 20:49:01 CET 2023
--
Errors in:
/usr/src/grass/lib/rst/interp_float
/usr/src/grass/lib/gpde
/usr/src/grass/raster/r.gwflow
/usr/src/grass/raster/r.mfilter
/usr/src/grass/raster/r.neighbors
/usr/src/grass/raster/r.patch
/usr/src/grass/raster/r.resamp.filter
/usr/src/grass/raster/r.resamp.interp
/usr/src/grass/raster/r.resamp.rst
/usr/src/grass/raster/r.series
/usr/src/grass/raster/r.series.accumulate
/usr/src/grass/raster/r.sim/simlib
/usr/src/grass/raster/r.sim/r.sim.water
/usr/src/grass/raster/r.sim/r.sim.sediment
/usr/src/grass/raster/r.slope.aspect
/usr/src/grass/raster/r.solute.transport
/usr/src/grass/raster/r.sun
/usr/src/grass/raster/r.univar
/usr/src/grass/raster3d/r3.gwflow
/usr/src/grass/vector/v.surf.bspline
/usr/src/grass/vector/v.surf.rst
```


```
my@DESKTOP-VADT8Q4 MINGW64 /usr/src/grass/lib/rst/interp_float
$ make
make lib
make[1]: Entering directory '/usr/src/grass/lib/rst/interp_float'
x86_64-w64-mingw32-gcc -shared -o /usr/src/grass/dist.x86_64-w64-mingw32/lib/libgrass_interpfl.8.3.dll -L/usr/src/grass/dist.x86_64-w64-mingw32/lib -L/usr/src/grass/dist.x86_64-w64-mingw32/lib -Wl,--export-dynamic,--enable-runtime-pseudo-reloc  -L/c/osgeo4w/lib   OBJ.x86_64-w64-mingw32/distance.o OBJ.x86_64-w64-mingw32/func2d.o OBJ.x86_64-w64-mingw32/init2d.o OBJ.x86_64-w64-mingw32/input2d.o OBJ.x86_64-w64-mingw32/interp2d.o OBJ.x86_64-w64-mingw32/matrix.o OBJ.x86_64-w64-mingw32/minmax.o OBJ.x86_64-w64-mingw32/output2d.o OBJ.x86_64-w64-mingw32/point2d.o OBJ.x86_64-w64-mingw32/resout2d.o OBJ.x86_64-w64-mingw32/ressegm2d.o OBJ.x86_64-w64-mingw32/secpar2d.o OBJ.x86_64-w64-mingw32/segmen2d.o OBJ.x86_64-w64-mingw32/segmen2d_parallel.o OBJ.x86_64-w64-mingw32/vinput2d.o OBJ.x86_64-w64-mingw32/write2d.o -lgrass_gis.8.3 -lintl -lgrass_raster.8.3 -lgrass_vector.8.3 -lgrass_gmath.8.3 -lgrass_dbmiclient.8.3 -lgrass_dbmibase.8.3  -lgrass_bitmap.8.3 -lgrass_qtree.8.3 -lgrass_interpdata.8.3   -lomp
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/12.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: warning: --export-dynamic is not supported for PE+ targets, did you mean --export-all-symbols?
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/12.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: OBJ.x86_64-w64-mingw32/segmen2d_parallel.o: in function `IL_interp_segments_2d_parallel._omp_fn.0':
C:/msys64/usr/src/grass/lib/rst/interp_float/segmen2d_parallel.c:108: undefined reference to `GOMP_loop_nonmonotonic_dynamic_start'
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/12.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/usr/src/grass/lib/rst/interp_float/segmen2d_parallel.c:143: undefined reference to `GOMP_loop_nonm
onotonic_dynamic_next'
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/12.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/usr/src/grass/lib/rst/interp_float/segmen2d_parallel.c:143: undefined reference to `GOMP_loop_end'
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/12.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/usr/src/grass/lib/rst/interp_float/segmen2d_parallel.c:384: undefined reference to `GOMP_critical_
start'
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/12.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/usr/src/grass/lib/rst/interp_float/segmen2d_parallel.c:384: undefined reference to `GOMP_critical_
end'
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/12.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: OBJ.x86_64-w64-mingw32/segmen2d_parallel.o: in function `IL_interp_segments_2d_parallel':
C:/msys64/usr/src/grass/lib/rst/interp_float/segmen2d_parallel.c:108: undefined reference to `GOMP_parallel'
collect2.exe: error: ld returned 1 exit status
make[1]: *** [../../../include/Make/Shlib.make:16: /usr/src/grass/dist.x86_64-w64-mingw32/lib/libgrass_interpfl.8.3.dll] Error 1
make[1]: Leaving directory '/usr/src/grass/lib/rst/interp_float'
make: *** [Makefile:15: default] Error 2

```

--- a/mswindows/osgeo4w/package.sh
+++ b/mswindows/osgeo4w/package.sh
@@ -133,7 +133,6 @@ DLLS="
 	/mingw64/bin/libfftw3-3.dll
 	/mingw64/bin/libblas.dll
 	/mingw64/bin/liblapack.dll
-	/mingw64/bin/libomp.dll
 	/mingw64/bin/libquadmath-0.dll
 "
 

